pkg/cmd/extension/manager_test.go

```diff
@@ -373,7 +414,8 @@ func TestManager_MigrateToBinaryExtension(t *testing.T) {
 	reg := httpmock.Registry{}
 	defer reg.Verify(t)
 	client := http.Client{Transport: &reg}
-	m := newTestManager(tempDir, &client, ios)
+	gc := &gitExecuter{client: &git.Client{}}
```

Comment:
This test uses `run.Stub()` which `git.Client` abides by, so there was no need to use the `mockGitClient` here.

Reply:
Hack 
```suggestion
	gc := &gitExecuter{client: &git.Client{}}
	
```